keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ApplicationEvent.kt

```diff
@@ -33,7 +33,7 @@ data class ApplicationActuationPaused(
   override val application: String,
   override val timestamp: Instant,
   override val triggeredBy: String?
-) : ApplicationEvent() {
+) : ApplicationEvent(), ResourceHistoryEvent {
```

Comment:
The new `ResourceHistoryEvent` interface provides a common type for events of any scope that belong in the resource event history.